src/modules/commander/PreflightCheck.cpp

```diff
@@ -345,7 +345,7 @@ static bool gyroCheck(orb_advert_t *mavlink_log_pub, unsigned instance, bool opt
 
 	if (ret != OK) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: GYRO #%u SELFTEST FAILED", instance);
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: GYRO #%u SELFTEST FAILED.", instance);
```

Comment:
Capitalization and wording should be updated for consistency with this PR: #10586
Suggested new text: "Preflight Fail: gyro #%u self test failed"